Source/OpenTK/Platform/X11/X11GLNative.cs

```diff
@@ -194,7 +194,9 @@ internal sealed class X11GLNative : NativeWindowBase
             hints.flags = (IntPtr)(XSizeHintsFlags.PSize | XSizeHintsFlags.PPosition);
 
             XClassHint class_hint = new XClassHint();
+#if !_NET_CORECLR
             var entry_assembly = Assembly.GetEntryAssembly();
```

Comment:
We do have a request to bring this one back: https://github.com/dotnet/corefx/issues/4146. I've usually seen this function being used in this way, i.e. just to get the name/identity of the entry assembly for diagnostic use, not really for any functional reason.